File: 디자인패턴/java 디자인 패턴/Bridge/CaptionDisplay.java
public class CaptionDisplay implements Display {
    public CaptionDisplay() {
        System.out.println("[Style - CaptionDisplay]");
    }
    @Override
    public void title(Draft draft) {
        System.out.println("Title: " + draft.getTitle());
    }
    @Override
    public void author(Draft draft) {
        System.out.println("Author: " + draft.getAuthor());
    }
    @Override
    public void content(Draft draft) {
        System.out.println("Content:");
        for (String item : draft.getContent()) {
            System.out.println("\t" + item);
        }
    }
}

File: 디자인패턴/Python 디자인 패턴/Bridge/CaptionDisplay.py
from Display import*
from Draft import*
class CaptionDisplay(Display):
    def __init__(self) -> None:
        print("[Style - CaptionDisplay]")
    
    def title(this, draft:Draft):
        print(f"Title: {draft.getTitle()}")

    def author(this, draft:Draft):
        print(f"Author: {draft.getAuthor()}")

    def content(this, draft:Draft):
        print("Content:")
        for item in draft.getContent():
            print(f"\t{item}")
    pass